internal/coreinternal/timeutils/internal/ctimefmt/ctimefmt.go

```diff
@@ -163,3 +163,14 @@ func Validate(format string) error {
 	}
 	return nil
 }
+
+func GetNativeSubstitutes(format string) map[string]string {
```

Comment:
```suggestion
// GetNativeSubstitutes analyzes the provided format string and returns a map where each
// key is a Go native layout element (as used in time.Format) found in the format, and
// each value is the corresponding ctime-like directive.
func GetNativeSubstitutes(format string) map[string]string {
```

Reply:
added the comment with the latest commit